src/udev/net/link-config.c

```diff
@@ -937,6 +938,61 @@ static int link_apply_sr_iov_config(Link *link, sd_netlink **rtnl) {
         return 0;
 }
 
+static int link_apply_rps_cpu_mask(Link *link) {
+        _cleanup_free_ char *mask_str, *dev_path, *rps_cpus_path = NULL;
+        _cleanup_closedir_ DIR *d = NULL;
+        LinkConfig *config;
+        int r;
+
+        assert(link);
+        config = ASSERT_PTR(link->config);
+
+        mask_str = cpu_set_to_mask_string(&config->rps_cpu_mask);
```

Comment:
this really should be a clean NOP if the option is not used. afaics you currently echo an empty string when the setting is not used into the sysfs attr, which is wrong...

Reply:
Now it will echo "0" if not set.